src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraVolumeMapperV1.kt

```diff
@@ -3,32 +3,80 @@ package no.skatteetaten.aurora.boober.mapper.v1
 import no.skatteetaten.aurora.boober.mapper.AuroraConfigFieldHandler
 import no.skatteetaten.aurora.boober.mapper.AuroraDeploymentSpec
 import no.skatteetaten.aurora.boober.model.AuroraConfigFile
+import no.skatteetaten.aurora.boober.model.AuroraSecret
 import no.skatteetaten.aurora.boober.model.AuroraVolume
 import no.skatteetaten.aurora.boober.model.Mount
 import no.skatteetaten.aurora.boober.model.MountType
+import no.skatteetaten.aurora.boober.utils.addIfNotNull
 import no.skatteetaten.aurora.boober.utils.ensureEndsWith
+import no.skatteetaten.aurora.boober.utils.ensureStartWith
 import no.skatteetaten.aurora.boober.utils.oneOf
 import no.skatteetaten.aurora.boober.utils.required
+import org.apache.commons.text.StringSubstitutor
 
-class AuroraVolumeMapperV1(private val applicationFiles: List<AuroraConfigFile>) {
+class AuroraVolumeMapperV1(
+    private val applicationFiles: List<AuroraConfigFile>,
+    val name: String,
+    val replacer: StringSubstitutor
```

Comment:
Hva er grunnen til at `replacer` sendes inn? Skal den være ulike i forskjellige bruksområder?

Reply:
For å tilate at man kan bruke @name@ placeholder overalt må vi replace alle verdier som _ikke_ hentes ut gjennom spec.get eller spec.getOrNull manuelt desverre :/